docs/related-work.md

```diff
@@ -9,7 +9,8 @@ Similar tools:
 * [Sapling](https://sapling-scm.com/): A heavily modified fork of
   [Mercurial](https://www.mercurial-scm.org/) developed and used at Meta. It
   is compatible with Git, has undo functionality, and
-  [a graphical interface](https://sapling-scm.com/docs/addons/isl). 
+  [a graphical interface](https://sapling-scm.com/docs/addons/isl).
+  See [how it is difference from Jujutsu](sapling-comparison.md).
```

Comment:
Typo: `how it is difference from Jujutsu` => `how it is different from Jujutsu` or `See the difference to Jujutsu`.

Reply:
Thanks, fixed